docs/sources/features/datasources/jaeger.md

```diff
@@ -0,0 +1,48 @@
++++
+title = "Using Jaeger in Grafana"
+description = "Guide for using Jaeger in Grafana"
+keywords = ["grafana", "jaeger", "guide", "tracing"]
+type = "docs"
+aliases = ["/docs/grafana/latest/datasources/jeager"]
+[menu.docs]
+name = "Jaeger"
+parent = "datasources"
+weight = 2
++++
+
+# Jaeger data source
+Grafana ships with built-in support for Jaeger, open source, end-to-end distributed tracing.
+Just add it as a data source and you are ready to query your traces in [Explore]({{< relref "../explore" >}}).
+
+## Adding the data source
+To access Jaeger settings, click the **Configuration** (gear) icon, then click **Data Sources**, and then click **Jaeger**.
+
+| Name            | Description                                                                                                                                   |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Name_          | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                                 |
+| _Default_       | Default data source means that it will be pre-selected for new panels.                                                                         |
+| _URL_           | The URL of the Jaeger instance, e.g., `http://localhost:16686`                                                                                   |
+| _Access_        | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser. |
+| _Basic Auth_    | Enable basic authentication to the Prometheus data source.                                                                            |
+| _User_          | User name for basic authentication.                                                                                                   |
+| _Password_      | Password for basic authentication.                                                                                                    |
+
+## Query traces
+
+Querying and displaying traces from Jaeger is available via [Explore]({{< relref "../explore" >}}).
```

Comment:
```suggestion
You can query and display traces from Jaeger via [Explore]({{< relref "../explore" >}}).
```

Reply:
Btw, this link will not work. The Explore topic is actually _index.md.

Reply:
Did something change? Because I copied it from Loki page https://github.com/grafana/grafana/blob/0e53b153758f5a4de3c4ec313f91e9ed36eb58a5/docs/sources/features/datasources/loki.md#L21 where it seems to work.